src/main/java/net/minecraftforge/event/level/LevelEvent.java

```diff
@@ -117,4 +125,78 @@ public ServerLevelData getSettings()
             return settings;
         }
     }
+
+    /**
+     * Called by NaturalSpawner to gather a list of all possible entities that can spawn at the specified location.
+     * If an entry is added to the list, it needs to be a globally unique instance.
+     * The event is called in {@link net.minecraft.world.level.NaturalSpawner#getRandomSpawnMobAt(ServerLevel,
+     * StructureManager, ChunkGenerator, MobCategory, RandomSource, BlockPos)}.
+     * Canceling the event will result in an empty list, meaning no entity will be spawned.
+     */
```

Comment:
Could you please reformat this to follow the general structure of the javadocs for client events (the `net.minecraftforge.client.event` package)?

Reply:
Is what I have in my latest push what you had in mind?